utopia-vscode-extension/snippets.json

```diff
@@ -29,5 +29,285 @@
       ")"
     ],
     "description": "Register Module Function Body"
+  },
+  "registerModule:string property control": {
+    "prefix": "registerModule:String Control",
+    "body": [
+      "${1:PropName}: {",
+      "  control: 'string-input',",
+      "  label: '${1:PropName}'",
+      "  //visibleByDefault: true",
+      "  //defaultValue: 'string'",
+      "  //placeholder: 'string'",
+      "  //obscured: false",
+      "},"
+    ],
+    "description": "Register Module String Control"
+  },
+  "registerModule:checkbox property control": {
+    "prefix": "registerModule:Checkbox Control",
+    "body": [
+      "${1:PropName}: {",
+      "  control: 'checkbox',",
+      "  label: '${1:PropName}'",
+      "  //defaultValue: false",
+      "  //visibleByDefault: true",
+      "  //disabledTitle: 'Disabled Title'",
+      "  //enabledTitle: 'Enabled Title'",
+      "},"
+    ],
+    "description": "Register Module Checkbox Control"
+  },
+  "registerModule:color property control": {
+    "prefix": "registerModule:Color Control",
+    "body": [
+      "${1:PropName}: {",
+      "  control: 'color',",
+      "  label: '${1:PropName}'",
+      "  //visibleByDefault: true",
+      "  //defaultValue: 'blue'",
+      "},"
+    ],
+    "description": "Register Module Color Control"
+  },
+  "registerModule:euler property control": {
+    "prefix": "registerModule:Euler Control",
+    "body": [
+      "${1:PropName}: {",
+      "  control: 'euler',",
+      "  label: '${1:PropName}'",
+      "  //visibleByDefault: true",
+      "  //defaultValue: [0, 0, 0, 'XYZ']",
+      "},"
+    ],
+    "description": "Register Module Euler Control"
+  },
+  "registerModule:none property control": {
+    "prefix": "registerModule:None Control",
+    "body": [
+      "${1:PropName}: {",
+      "  control: 'euler',",
+      "  label: '${1:PropName}'",
+      "  //visibleByDefault: true",
```

Comment:
```suggestion
```
The `none` control is to prevent a control from being shown for that prop, so the snippet shouldn't include these two lines